### PR TITLE
Feature/user signs in

### PR DESCRIPTION
**Pull request by aakyol**
Hi all. Since we are done with sign in, VJ said we should do a PR on it, hence my request.

### CODE REVIEW

**Review by AigerimSerikbekova — APPROVED** [review]

**Review by spiroskous — APPROVED** [review]

File: android_client/ChitChat/app/src/main/java/com/moc/chitchat/activity/LoginActivity.java [hunk 1]
```diff
@@ -51,20 +66,64 @@ public void onClick(View v) {
...
     @Override
     public void onErrorResponse(VolleyError error) {
+        System.out.println("Error registering");
+
+        try {
 
+            Toast.makeText(this,
+                String.format("Invalid credentials or you didn't registered yet"),
+                Toast.LENGTH_LONG).show();
+        } catch (Exception e) {
```
Comment (VJftw) [inline android_client/ChitChat/app/src/main/java/com/moc/chitchat/activity/LoginActivity.java]: @aakyol The try/catch here is redundant and so is the String.format :) So this bit could just be:
```
Toast.makeText(this,
            "Invalid credentials or you didn't registered yet",
            Toast.LENGTH_LONG
        ).show();
```

File: backend/chit_chat/config/config.exs [hunk 2]
```diff
@@ -22,6 +22,17 @@ config :logger, :console,
...
 
+# Configures Guardian JWT auth system
+config :guardian, Guardian,
+  allowed_algos: ["HS512"],
+  verify_module: Guardian.JWT,
+  issuer: "ChitChat",
+  ttl: { 30, :days },
+  allowed_drift: 2000,
+  verify_issuer: true,
+  secret_key: "cQCWxKpcuixeB4ZAxCs04nrBGdKeJiHcmmCHbZPI6esGcLcfZVz1qw2796p3gWGA",
```
Comment (VJftw) [inline backend/chit_chat/config/config.exs:33]: I need to extract this into an environment variable for use in the docker containers.

File: java_client/ChitChatDesktop/test/com/moc/chitchat/application/ApplicationLoaderTest.java [hunk 3]
```diff
@@ -13,61 +13,61 @@
...
-
-        ApplicationLoader applicationLoader = new ApplicationLoader(mockSplashWindow, mockRegistrationView, mockConfiguration);
-
-        String[] args = {"dev"};
-
-        applicationLoader.load(args);
-
-        verify(mockSplashWindow).setVisible(true);
-    }
+//    @Test
```
Comment (VJftw) [inline java_client/ChitChatDesktop/test/com/moc/chitchat/application/ApplicationLoaderTest.java]: Need to fix these tests

File: java_client/ChitChatDesktop/test/com/moc/chitchat/application/SplashWindowTest.java [hunk 4]
```diff
@@ -1,73 +0,0 @@
-package com.moc.chitchat.application;
```
Comment (VJftw) [inline java_client/ChitChatDesktop/test/com/moc/chitchat/application/SplashWindowTest.java:1]: Remove the SplashWindow and SplashWindowTest as it's not being used

**Review by VJftw — CHANGES_REQUESTED** [review]
@aakyol just the one android change and i'll do the others for the backend and java client as i initiated those and should have cleaned up after myself :p

File: android_client/ChitChat/app/src/main/java/com/moc/chitchat/activity/LoginActivity.java (hunk 1 above)
Comment (aakyol) [reply android_client/ChitChat/app/src/main/java/com/moc/chitchat/activity/LoginActivity.java]: Changing it from the browser now.

File: java_client/ChitChatDesktop/test/com/moc/chitchat/application/ApplicationLoaderTest.java (hunk 3 above)
Comment (VJftw) [reply java_client/ChitChatDesktop/test/com/moc/chitchat/application/ApplicationLoaderTest.java]: Fixed.

File: java_client/ChitChatDesktop/test/com/moc/chitchat/application/SplashWindowTest.java (hunk 4 above)
Comment (VJftw) [reply java_client/ChitChatDesktop/test/com/moc/chitchat/application/SplashWindowTest.java:1]: Fixed.

**Review by Ozhan-Azizi — APPROVED** [review]

**Review by FareedQasim — APPROVED** [review]

**Review by VJftw — APPROVED** [review]

**Pull request merged**